docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md

```diff
@@ -50,6 +52,10 @@ When the Biome REST API receives the authorization request for Alice, it fetches
 the entry from the Acme Splinter daemon's local database associated with the
 username and verifies the hashed password sent in the request.
 
+
```

Comment:
This images still refer to the gameroom REST API, but the text mentions Biome.  Seems like there is a missing box here.

Reply:
A missing box in the image? The box to the right of the Splinter daemon in the diagram represents the Biome REST API, I felt as though that was a sufficient representation.

Reply:
I still see game room daemon on the right, which is confusing, considering the text. Shouldn’t that be splinterd?